documentation/blog/2023-08-01-whats-new-in-svelte-august-2023.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "What's new in Svelte: August 2023"
+description: "Extending Custom Elements with CSS and new +server customizations"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Some sweet new features have dropped in both Svelte and SvelteKit, this month. It's also great to see how many products and side projects have launched using Svelte as their core technology!
+
+More on all that down below...
+
+## What's new in Svelte & Language Tools
+There's been a bunch of minor bugfixes since the Svelte 4 release. You can find them in the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md).
+
+In addition, the **4.1.0 release** added the ability to extend custom element classes. Check out the [Custom Elements API docs](https://svelte.dev/docs/custom-elements-api) or the [PR](https://github.com/sveltejs/svelte/pull/8991) for more info!
+
+In addition to supporting SvelteKit's new `HEAD` server method, Svelte's language tools now support Prettier v3 (**extensions-107.9.0**) and workspace trust settings are now used to support all settings in workspace (**extensions-107.8.0**).
```

Comment:
```suggestion
Svelte's language tools got updated to support SvelteKit's new `HEAD` server method and Prettier v3 (**extensions-107.9.0**). Also workspace trust settings are now used to support all settings in workspace (**extensions-107.8.0**).
```
Maybe we can refrase to avoid the "In addition" repetition?